stdlib.go

```diff
@@ -79,7 +81,13 @@ func TAB(val int) {
 
 func CLS() {
 	// see https://stackoverflow.com/a/22892171
-	fmt.Print("\033[H\033[2J")
+	if runtime.GOOS == "windows" {
+		cmd := exec.Command("cmd", "/c", "cls")
+        cmd.Stdout = os.Stdout
```

Comment:
Tabs vs space issue here. Line 86 and 87.

Reply:
Oops - didn't pay attention to that. Committed a fix.